html/elements/input/color.json

```diff
@@ -136,7 +136,7 @@
                   "version_added": null
                 },
                 "safari_ios": {
-                  "version_added": false
+                  "version_added": null
```

Comment:
We typically discourage changes from `true/false` or a version number **to** null, especially as it acts against the 2019 Key Result of removing all `null` and `true` values.  But for this one, I'm seeing possible signs that it's actually supported.  I'd like to get some more research on the manner, so as such, I'm going to accept this one.